cpp/velox/substrait/SubstraitToVeloxExpr.cc

```diff
@@ -519,19 +519,98 @@ RowVectorPtr SubstraitVeloxExprConverter::literalsToRowVector(const ::substrait:
   if (childSize == 0) {
     return makeEmptyRowVector(pool_);
   }
-  auto typeCase = structLiteral.struct_().fields(0).literal_type_case();
-  switch (typeCase) {
-    case ::substrait::Expression_Literal::LiteralTypeCase::kBinary: {
-      std::vector<VectorPtr> vectors;
-      vectors.reserve(structLiteral.struct_().fields().size());
-      for (auto& child : structLiteral.struct_().fields()) {
+  std::vector<VectorPtr> vectors;
+  vectors.reserve(structLiteral.struct_().fields().size());
+  for (auto& child : structLiteral.struct_().fields()) {
```

Comment:
Not your change. Better to use `const &`.

Reply:
Updated, thanks.